scripts/garage.coffee

```diff
@@ -41,17 +49,31 @@ module.exports = (robot) ->
 	getGarages = (callback) ->
 		r = request 'https://secure.parking.ucf.edu/GarageCount/iframe.aspx/', (error, response, body) ->	
 			garages = []
-			$ = cheerio.load(body);
+			pubsub = false;
+			$ = cheerio.load(body)
 			$('.dxgvDataRow_DevEx').each (i, obj) ->
-				thisGarage = {};
+				thisGarage = {}
 				html = $(obj).html().replace(RegExp(' ', 'g'), '').split '\n'
 				for line in html
 					if line.startsWith("percent:")
 						percent = parseInt(line.replace("percent:", ''))
 						thisGarage.perc = percent
 				thisGarage.garage = ($(obj).find('.dxgv').html()).replace("Garage ", '')
 				garages[i] = thisGarage
-			callback garages
-		return null
-
-
+			
+			if robot.brain.get('pubsubCheckDate') isnt (new Date()).toDateString()
```

Comment:
Since now we have two different places where we do the same thing (get pubsub stuff) it would probably be best to have a function somewhere else that is used both here and in [scripts/pubsub.js](https://github.com/SammyIsra/fred/blob/master/scripts/pubsub.js). 

Buuut we may want to hold off this specific change until we deal with #24